src/lumigo_tracer/utils.py

```diff
@@ -15,10 +16,11 @@
 LUMIGO_EVENT_KEY = "_lumigo"
 STEP_FUNCTION_UID_KEY = "step_function_uid"
 MAX_SIZE_FOR_REQUEST: int = int(os.environ.get("LUMIGO_MAX_SIZE_FOR_REQUEST", 900_000))
-MAX_VARS_SIZE = 100000
+MAX_VARS_SIZE = 100_000
 MAX_VAR_LEN = 200
 MAX_ENTRY_SIZE = 1024
 FrameVariables = Dict[str, str]
+OMITTING_KEYS_REGEXES = [".*pass.*", ".*key.*"]
```

Comment:
Does the user have an ability to override these?

Reply:
no.
we should give him that?
that sounds like complicated docs...

Reply:
I just think 
`OMITTING_KEYS_REGEXES + json.loads(os.environ.get("LUMIGO_BLACKLIST_REGEX", "[]"))`
Should not be a `+`if the env is empty then use the default

Reply:
hmm...
fair, I'll change it :)